### ISSUE TITLE
FMP Claims - Staging Review finding: Heading structure of file upload page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99261]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-7959f-2, Foreign Medical Program claim cover sheet
**Feature:** 

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/file-claim-form-10-7959f-2/upload-supporting-documents](https://staging.va.gov/health-care/foreign-medical-program/file-claim-form-10-7959f-2/upload-supporting-documents), the first heading of the page content ("Upload billing statements and supporting documents") is an H2 heading. This breaks the heading pattern of the form flow and VA forms in general. The step indicator is itself an H2 and describes the page of the form, and any headings within the page content would then begin at H3.

Screen reader users often navigate a page by heading, and the level announced for each heading provides useful structural information about the page. When a heading level is misused or breaks the pattern that's been in place for the flow of the form up to this point, it can be confusing and even disorienting.

<details><summary>Link, screenshot or steps to recreate</summary>

![Step 7 of 8 Supporting Files, with an H2 heading "Upload billing statements and supporting documents" followed by the remainder of the page content.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/77082-Brian-2.png?raw=true)

</details>

## Recommended action

"Upload billing statements..." should be an H3, and the headings that follow ("Supporting documents..." and "How to upload files") should be H4 headings.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

--

## Acceptance Criteria
- [ ] Update form to use the new minimal header, which will address this issue
- [ ] Tests added/updated
- [ ] Ticket created for UX team to update source of truth
